app/src/main/java/org/p2p/wallet/user/repository/prices/impl/TokenPricesCoinGeckoRepository.kt

```diff
@@ -15,35 +15,25 @@ class TokenPricesCoinGeckoRepository(
     override suspend fun getTokenPriceByIds(
         tokenIds: List<TokenId>,
         targetCurrency: String
-    ): List<TokenPrice> = withContext(dispatchers.io) {
-        loadPrices(tokenIds, targetCurrency)
-    }
+    ): List<TokenPrice> = loadPrices(tokenIds, targetCurrency)
 
     override suspend fun getTokenPriceById(
         tokenId: TokenId,
         targetCurrency: String
-    ): TokenPrice = withContext(dispatchers.io) {
-        loadPrices(listOf(tokenId), targetCurrency).first()
-    }
+    ): TokenPrice = loadPrices(listOf(tokenId), targetCurrency).first()
```

Comment:
it's crash otherwise _)

Reply:
why? loadPrices inside wrapped in withContext too

Reply:
oh, I just meant that, `first()` fun will throw exception)

Reply:
yes, it will, but I haven't added first, just removed with context